packages/commercetools/api-client/__tests__/helpers/tokenFlow.spec.ts

```diff
@@ -1,30 +1,32 @@
 /* eslint-disable camelcase, @typescript-eslint/camelcase */
-import createAccessTokenFlow from './../../src/helpers/createCommerceToolsLink/tokenFlow';
-
-jest.mock('@commercetools/sdk-auth', () => jest.fn(() => ({
-  anonymousFlow: () => Promise.resolve('anonymous flow'),
-  customerPasswordFlow: () => Promise.resolve('customer password flow'),
-  refreshTokenFlow: () => Promise.resolve('refresh token flow')
-})));
-
-const apiConfig = {
-  authHost: 'localhost',
-  projectKey: 'project-key',
-  clientId: 'client-id',
-  clientSecret: 'client-secret',
-  scopes: []
+import createAccessToken from './../../src/helpers/createAccessToken';
+import { setup } from './../../src/index';
+
+const config = {
+  api: {
+    uri: 'https://example.com',
```

Comment:
It's a good practice to provide non-existing URLs so nothing weird will go "outside" when things have gone wrong, for example: `https://localhost` or just `https://github.com/DivanteLtd/next`

Reply:
+1, you can also just hardcode our demo instance ;)

Reply:
it can't go outside, there is a mock underneath